e2e/init.js

```diff
@@ -3,6 +3,8 @@
 require('dotenv').config({ path: '.env' });
 
 beforeAll(async () => {
+  await device.clearKeychain();
```

Comment:
This is quite a big change, so just to check if this is what you mean to do (I don't see any reason not to).

Reply:
yeah i saw the app starting again with an old keychain, `device.clearKeychain` from `afterAll` doesn't always run it seems. think this improves the e2e flow when one fails